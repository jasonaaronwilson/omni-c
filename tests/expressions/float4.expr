1.23e-5;
