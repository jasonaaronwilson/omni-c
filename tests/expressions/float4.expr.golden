{
    tag = PARSE_NODE_LITERAL
    tokens = [
        1.23e-5
    ]
}

// C Output
1.23e-5
